tslint.json

```diff
@@ -1,6 +1,7 @@
 {
-  "extends": ["tslint:latest", "tslint-config-prettier"],
+  "extends": ["tslint:latest", "tslint-consistent-codestyle"],
```

Comment:
Can we have both tslint-config-prettier and tslint-consistent-codestyle? We use prettier on all the graphile projects, and I would like to keep it that way. (Trailing commas on function arguments can be enabled by changing `trailingComma: "es5"` to `trailingComma: "all"` in the prettier config.)